sdk/identity/identity-broker/samples/v1-beta/typescript/README.md

```diff
@@ -0,0 +1,80 @@
+---
+page_type: sample
+languages:
+  - typescript
+products:
+  - azure
+  - azure-active-directory
+  - WAM
+  - MSA support
+urlFragment: identity-broker-typescript-beta
+---
+
+# Azure Identity Brokered Auth Plugin client library samples for TypeScript (Beta)
+
+These sample programs show how to use the TypeScript client libraries for Azure Identity Brokered Auth Plugin in some common scenarios.
+
+| **File Name**     | **Description**       |
+| ----------------- | --------------------- |
+| [index.ts][index] | Demonstrates usage of |
+
+## Prerequisites
+
+The sample programs are compatible with [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule).
+
+Before running the samples in Node, they must be compiled to JavaScript using the TypeScript compiler. For more information on TypeScript, see the [TypeScript documentation][typescript]. Install the TypeScript compiler using:
+
+```bash
+npm install -g typescript
+```
+
+You need [an Azure subscription][freesub] and the following Azure resources to run these sample programs:
+
+- [Azure Active Directory App Registration][createinstance_azureactivedirectoryappregistration]
```

Comment:
I think this needs to change to Microsoft Entra to confirm with the branding update.

/cc @scottaddie

Reply:
done